src/components/Icons.js

```diff
@@ -1,6 +1,9 @@
-import { ReactComponent as Check } from '../assets/check.svg'
-import { ReactComponent as Cross } from '../assets/cross.svg'
-import { ReactComponent as Bin } from '../assets/bin.svg'
-import { ReactComponent as Pencil } from '../assets/pencil.svg'
+import { ReactComponent as Check } from '../assets/icons/check.svg'
+import { ReactComponent as Cross } from '../assets/icons/cross.svg'
+import { ReactComponent as Bin } from '../assets/icons/bin.svg'
+import { ReactComponent as Pencil } from '../assets/icons/pencil.svg'
+import { ReactComponent as Plus } from '../assets/icons/plus.svg'
+import { ReactComponent as Home } from '../assets/icons/home.svg'
+import { ReactComponent as Logo } from '../assets/icons/golf-logo.svg'
```

Comment:
Why do you have this specific Icon.js?
you could also import them right where you need them...
...just curious...

Reply:
Was a comment of a coach in a code review in our course. Doing this, all icons are grouped together and you do not have to write ``ReactComponent`` for importing the icons in your working files anymore. :)

Reply:
ahh interesting- we actually write it without "ReactComponent"....just like this:
```import Home from '.../assets/home.svg' ```